src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3664,6 +3667,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return _core.OwningHwnd();
     }
 
+    void TermControl::PreviewInput(const winrt::hstring& text)
+    {
+        get_self<ControlCore>(_core)->PreviewInput(text);

```

Comment:
There should be an a11y notification event here since we're displaying new content to the screen. The screen reader doesn't pick up on it.

Something like this, but modify as you'd like:
```suggestion
        get_self<ControlCore>(_core)->PreviewInput(text);
        if (auto automationPeer{ FrameworkElementAutomationPeer::FromElement(*this) })
        {
            automationPeer.RaiseNotificationEvent(
                AutomationNotificationKind::ItemAdded,
                AutomationNotificationProcessing::All,
                text, // what to announce
                L"TSFPreviewInputAnnouncement" /* unique name for this group of notifications */);
        }
```